pkgs/top-level/python-packages.nix

```diff
@@ -10510,15 +10510,24 @@ in modules // {
 
   flake8 = buildPythonPackage rec {
     name = "flake8-${version}";
-    version = "2.5.4";
+    version = "3.0.4";
 
     src = pkgs.fetchurl {
       url = "mirror://pypi/f/flake8/${name}.tar.gz";
-      sha256 = "0bs9cz4fr99r2rwig1b8jwaadl1nan7kgpdzqwj0bwbckwbmh7nc";
+      sha256 = "03cpdrjxh0fyi2qpdxbbrmxw7whiq3xr3p958gr6yzghk34i1hml";
     };
 
     buildInputs = with self; [ nose mock ];
```

Comment:
Nice catch! I think it works on my machine because it uses the system pytest/pytesrunner. 

While creating the PR i read the first time about the sandbox mode. Didn't find very much information about it. Is it possible to run my changes in sandbox mode on OSX or is it limited to NixOS?